frontend/src/components/blocks/SearchResult.tsx

```diff
@@ -56,30 +42,20 @@ const SearchResult = ({ document }: SearchResultProps) => {
       <div className="flex justify-between items-start">
         <h2 className="leading-none flex items-center">
           <Link href={`/document/${document.document_id}`}>
-            <a className="text-left font-medium text-lg leading-tight">
-              {truncateString(document.document_name, 80)}
-            </a>
+            <a className="text-left font-medium text-lg leading-tight">{truncateString(document.document_name, 80)}</a>
           </Link>
         </h2>
       </div>
 
       <div className="flex flex-wrap lg:flex-nowrap text-sm text-indigo-400 mt-4 items-center">
-        <div
-          className={`rounded-sm border border-black flag-icon-background flag-icon-${document.document_country_code.toLowerCase()}`}
-        />
+        <div className={`rounded-sm border border-black flag-icon-background flag-icon-${document.document_country_code.toLowerCase()}`} />
         <span className="ml-2">
-          {document.document_country_english_shortname}
+          {document.document_country_english_shortname}, {year}
```

Comment:
Does this also need to be labelled as a future candidate for translation as above?

Reply:
Yes true, dynamic translations need to be considered. So far we have some limited static translations for basic copy but things like countries and other taxonomies are just rendered as English.